src/plugins/port/README.md

```diff
@@ -0,0 +1,27 @@
+## Introduction
+
+This plugin is a check plugin that checks if the port on localhost is open and a valid number.
+
+## Purpose
+
+One source of misconfiguration can come from invalid port numbers or from taking used ports such as 22 for ssh.
+This plugin checks if the given port is open as well as the given port lies between 1-65535
+
+## Usage
+
+If `check/port` is specified on a given key, the plugin will validate if the port is a 
+correct number between 1 and 65535.
+Furthermore it will check if the port is open to use.
+
+If `check/simpleport` is specified, only the correct port range will be checked. The port may be in use which is 
```

Comment:
Indeed, but I didn't come up with a better idea after pondering about this to be honest :P . I will change the behavior as follows:

`check/port/open` Range + available
`check/port/range` Range

I think this is better

Reply:
What about:

`check/port` for the range check (and/or transformation from /etc/services)
`check/port/listen` to check if the port can be listened to (usually for servers)
`check/port/ping` to check if the port can be pinged/reached  (usually for clients)

Reply:
`check/port/ping` sounds promising but I also think that we first should gather data about the usage. Imo it is often the case the client and server configurations are set before the server is actually started (for example I do it for LCDproc like this). This would prohibit the user from setting a potentially correct setting.

I will add it to a potential future work section

Reply:
It is also possible and useful to emit warnings.

> I will add it to a potential future work section

If you think it does not work well, it would be rather in the "Discarded Ideas" section.